### PR TITLE
Add mobile specific layout and mitigation panel

### PR DESCRIPTION
**Pull request by McPenquen**

### CODE REVIEW

File: frontend/src/app/game/game/game.component.ts [hunk 1]
```diff
@@ -51,11 +60,23 @@ export class GameComponent {
...
+
   @HostBinding('class.is-event-active')
   get isEventActiveClass() {
     return this.gameService.event;
   }
 
+  @HostListener('window:resize', ['$event'])
+  onResize(event: any) {
+    this.winWidth = event.target.innerWidth;
+    if (this.winWidth <= this.maxMobileWidth) {
```
Comment (zdenekhatak) [inline frontend/src/app/game/game/game.component.ts:72]: this.isSmallDevice = this.winWidth <= this.maxMobileWidth;

File: frontend/src/app/game/game/game.component.scss [hunk 2]
```diff
@@ -25,3 +25,11 @@
 .content {
   width: 100%;
 }
+
+.mobile-mitigation-panel {
+  position: fixed;
+  width: 100vw;
+  max-height: fit-content;
+  bottom: -0.5%;
+  background-color: #222222;
```
Comment (jnwltr) [inline frontend/src/app/game/game/game.component.scss:34]: Pls, do not use magic constants, we have e.g. `frontend/src/material-definitions.scss`.

File: frontend/src/app/game/game/game.component.ts [hunk 3]
```diff
@@ -18,6 +19,9 @@ type GameState = 'intro' | 'game' | 'outro';
 export class GameComponent {
 
   state: GameState = 'intro';
+  isSmallDevice: boolean = false;
+  winWidth: number = window.innerWidth;
+  maxMobileWidth: number = 894;
```
Comment (jnwltr) [inline frontend/src/app/game/game/game.component.ts:24]: Should not we have it as a constant somewhere?

**Comment by jnwltr** [comment]
Unfinished work.

**Pull request closed without merging**